src/components/HomeMain.component/Categories.component/HomeCategories.jsx

```diff
@@ -0,0 +1,16 @@
+import racket from "../../../Assets/images/racquet.png";
+import React from "react";
+import "./home-categories.css";
+
+const HomeCategories = () => {
+  return (
+    <div className="cat-card br-sm">
+      <img className="res-img" src={racket} alt="" />
+      <a className="button btn-sm cat-card-btn text-sm" href="">
```

Comment:
Don't keep the alt attribute empty.

Reply:
Its not the final code. react router is left to be implemented. i'll take care of that